src/anm/animation/tween.js

```diff
@@ -259,7 +260,7 @@ Tween.register(C.T_ROT_TO_PATH, {
             var path = this.$mpath;
             // when t equals exact 0, it is replaced with 0.001
             // or else returned angle would be 0
-            if (path) this.angle += path.tangentAt(t || 0.001);
+            if (path) this.angle += path.tangentAt(this.$mpath_t || 0.001);
```

Comment:
Honestly, it's also not about static parts :) static parts are skipped [here](https://github.com/Animatron/player/pull/322#discussion-diff-34006892R221) instead. It's just because tangent at the very start point of any path is always 0.